components/validation/src/main/java/org/wso2/carbon/identity/x509Certificate/validation/util/CertificateValidationUtil.java

```diff
@@ -721,32 +522,6 @@ private static void getAllTrustedCerts(Iterator trustStoreIterator, List<X509Cer
      * ****************************************
      */
 
-    /**
-     * Extracts all CRL distribution point URLs from the "CRL Distribution Point" extension in a X.509 certificate.
-     * If CRL distribution point extension or CRL Urls are unavailable, throw an exception.
-     *
-     * @param cert X509 certificate
-     * @return List of CRL Urls in the certificate
-     * @throws CertificateValidationException certificateValidationException
-     */
-    public static List<String> getCRLUrls(X509Certificate cert) throws CertificateValidationException {
```

Comment:
Removal of a public method.. Double check the usage.

Reply:
Not removed, but refactored to a different class (X509CertificateUtil).
Shall I keep those in here.

Reply:
Less critical than the comment https://github.com/wso2-extensions/identity-x509-commons/pull/56#issuecomment-2635955697.. since this will not break the extension point's contract..

But, this will effect the implementation, so better to keep this, if not affect the design.